cmd/cli/log.go

```diff
@@ -0,0 +1,69 @@
+package cli
+
+import (
+	"fmt"
+
+	logging "github.com/ipfs/go-log/v2"
+	"github.com/spf13/cobra"
+	"github.com/storacha/piri/cmd/cli/client"
+)
+
+var logListCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List all logging subsystems and their levels",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		// List all registered subsystems
+		subsystems := logging.GetSubsystems()
+		for _, subsystem := range subsystems {
+			level := logging.Logger(subsystem).Level().String()
+			fmt.Printf("%-30s %s\n", subsystem, level)
+		}
+		return nil
+	},
+}
+
+var logSetLevelCmd = &cobra.Command{
```

Comment:
Actually, I've been thinking about this more and want to suggest a different approach:

`piri log set-level <level>` would set the level across all logging systems, while `piri log set-level --system <system1> --system <system2> <level>` would target specific systems. You can pass multiple `--system` flags to configure several systems at once.
This feels cleaner and gives users the flexibility to be either broad or precise depending on what they need. For the implementation, `StringSlice` would work better than `StringArray` for the flag handling.
What do you think about this direction?